extensions/kafka/src/main/java/io/deephaven/kafka/ingest/GenericRecordChunkAdapter.java

```diff
@@ -120,7 +124,12 @@ private static FieldCopier makeFieldCopier(
                             "Can not map field with non matching logical type to BigDecimal: " +
                                     "field=" + fieldPathStr + ", logical type=" + logicalType);
                 }
-                return new GenericRecordObjectFieldCopier(fieldPathStr, separator, schema);
```

Comment:
Is there no possibility of hitting a bare `Object` that isn't a `GenericContainer` now?

Reply:
No, that shouldn't be possible, AFAIK the kafka deserializer wouldn't be producing that.

Reply:
Reworked this.